tests/settings.py

```diff
@@ -84,6 +84,7 @@
     }
 }
 
+CUSTOM_ADMIN_SITE = 'openwisp_utils.admin_theme.admin.OpenwispAdminSite'
```

Comment:
please define a custom site class in the `test_project`, which extends `OpenwispAdminSite` and adds a custom attribute, then write a test which asserts the presence of this custom attribute.

Use this settings file to specify the path to the custom admin site class.